SimPEG/electromagnetics/static/utils/StaticUtils.py

```diff
@@ -354,10 +324,8 @@ def plot_pseudoSection(
     if dobs is None:
         dobs = data.dobs
 
-
-    midx, midz = source_receiver_midpoints(
-        data.survey, survey_type=survey_type, dim=dim
-    )
+    midx, midz = source_receiver_midpoints(data.survey)
+    midx = midx[:, 0]  # Only keep the along line
```

Comment:
I don't like that this assumes every line is run east-west in 3D. Something should be added to check for the orientation of the lines, and choose x or y in 3D.